### ISSUE TITLE
FlowFilters added to Card Pages don't works and don't update FlowFields

### ISSUE THREAD

**Issue by JavierFuentes** [microsoft/AL#4457]
**Describe the bug**
Actually seems we can't manage Filters and FlowFilters in Card Pages as we can do in List Pages...

So, we have tried to add FlowFilters and FlowFields directly into a Card Page Extension, but it doesn't works.

**To Reproduce**
Add a FlowFilter and a FlowField to a Card Page or a Card Page Extension.

When you validate a new value into a FlowFilter, it doesn't remains in the field and seems it lost its value... as a consequence, the FlowField don't update its value neither.

```php
pageextension 50100 prefix_GLAccountCardExt extends "G/L Account Card"
...
field("prefix_Date Filter"; "Date Filter")
{
   ApplicationArea = All;

   trigger OnValidate()
   begin
       CalcFields("Balance at Date"); // Forze code calcfield don't works neither
       Message('The new Date Filter is: %1', GetFilter("Date Filter"));
   end;
}

field("prefix_Balance at Date"; "Balance at Date")
{
   ApplicationArea = All;
}
```

Example: 
[FlowFilterInCardsIssue.zip](https://github.com/Microsoft/AL/files/2753020/FlowFilterInCardsIssue.zip)

**Expected behavior**
Ideally we need to interact with Filters and FlowFilters in Cards nativelly...

Regardless of whether this works at any time, we need to be able to add fields of type flowfilter and flowfield to the Page Card and that they work as expected.

**Screenshots**
![image](https://user-images.githubusercontent.com/5801847/51084683-71ff4500-172e-11e9-9e3d-d35dd169ebf0.png)

**Versions**:
AL Language: **2.0.48254**
Business Central: **ES Dynamics NAV 13.0 (24630) OnPremise**

**Comment by JavierFuentes** [comment]
We have noticed that if you validate the "Date Filter" value in "G/L Account List" this value is copied to "G/L Account Card" and used to calculate the "Balance at Date" FlowField, but *the FlowFilter added to Page don't show this value*

In List:
![image](https://user-images.githubusercontent.com/5801847/51084877-9c9ecd00-1731-11e9-9f1a-4f7c2556c6c5.png)

In Card:
![image](https://user-images.githubusercontent.com/5801847/51084878-a294ae00-1731-11e9-86f3-923a7f188668.png)

**Comment by JohanStenberg100** [comment]
Hi @JavierFuentes, thanks for filing this issue, we'll treat this as a bug and will try to fix it.

**Comment by JavierFuentes** [comment]
Hello,
**the error reported in this issue is still not fixed in the latest version of BC available as of this date (v19.3)**

Please reopen this same ticket to try to fix it... i spent too much effort documenting the problem to open a new one 25 months later...

Thanks in advance.

**Comment by mspindev** [comment]
Hey everyone, you should set the flow filter with SetRange or SetFilter and not assign it with := or validate. Then, if necessary, a Calcfields() with the FlowField.
